docs/data/charts/tooltip/tooltip.md

```diff
@@ -90,6 +90,24 @@ In this demo, you can see:
 
 {{"demo": "AxisFormatter.js"}}
 
+### Label formatting
+
+The label text inside the tooltip can also be formatted conditionally by providing a function to the series' `label` property.
```

Comment:
Both are correct, since this is somewhat a "weak link". Technically series owns the label, so `series' label` is "more" correct, your suggestion is simpler though, and most ppl wont find it odd. 😅

Reply:
Yeah, those are great observations, I completely agree with everything, but simplicity always wins if it doesn't impede clarity. 👍